collectivites/templates/index.html

```diff
@@ -4,9 +4,83 @@
 
 {% block content %}
 <div class="row mwide twide">
-    <div class="col wide">
-
-        <p>Bienvenue sur le portail collectivités blah…</p>
-    </div>
+    <main class="col wide">
+        <ban></ban>
+        <territory></territory>
+    </main>
 </div>
+<script type="riot/tag" src="{{ url_for('static', filename='tags/ban.js') }}"></script>
+<script type="riot/tag" src="{{ url_for('static', filename='tags/territory.js') }}"></script>
+<script>
+    const API_URLS = {{ API_URLS|tojson }}
+    const defaultCenter = [47, 2]
+    const map = L.map('map', {center: defaultCenter, zoom: 5, zoomControl: false})
+    const tilelayer = L.tileLayer('http://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png').addTo(map)
+    const onEachFeature = (feature, layer) => {
+        layer.on('click', () => {
+            territories[feature.id] = feature
+            riot.route(`/territoire/${feature.id}`)
+        })
+    }
+    const group = L.geoJson(null, {
+        onEachFeature: onEachFeature
+    }).addTo(map)
+    new L.PhotonBaseSearch(L.DomUtil.get('search-territory'), {
+        url: `${API_URLS.GEOZONES}?`,
+        placeholder: 'Chercher une commune, un département, une région',
+        onSelected: (feature) => {
+            riot.route(`/territoire/${feature.id}`)
+        }
+    });
+
+    L.DomEvent.on(Z.qs('#geolocate'), 'click', () => {
+        navigator.geolocation.getCurrentPosition(
+            (position) => {
+                fetch(`${API_URLS.GEOAPI}communes?lat=${position.coords.latitude}&lon=${position.coords.longitude}`)
+                    .then((response) => {
+                        return response.json()
+                    }).then((json) => {
+                        const codeInsee = json[0].codeInsee
```

Comment:
Bah, on a confiance dans la geoAPI ;)
Plus sérieusement: mettre un test clairement, mais du coup aussi gérer le cas où ça passe pas, et donc en informer l'utilisateur. Bref, un système de messages is needed :)